nf_core/modules/lint/module_tests.py

```diff
@@ -22,6 +22,8 @@ def module_tests(_, module: NFCoreComponent):
     and contains a ``main.nf.test`` and a ``main.nf.test.snap``
 
     """
+    if module.nftest_testdir is None or module.nftest_main_nf is None:
+        raise ValueError()
```

Comment:
don't we want to allow local modules to not have these files?

Reply:
Yes, this was my first-pass to get `mypy` happy so I could run the tests on GitHub yesterday. I am updating the individual lint tests to be a bit more permissive of missing files for local modules.